packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol

```diff
@@ -388,6 +388,9 @@ interface ISuperToken is ISuperfluidToken, TokenInfo, IERC20, IERC777 {
      *
      * @custom:note It will use `transferFrom` to get tokens. Before calling this
      * function you should `approve` this contract
+     * 
+     * @custom:requirements 
```

Comment:
:+1:

Reply:
add a @custom:warning about the potential re-entrance case if the "to" account is ERC-777.

Reply:
resolved mr